packages/react/src/components/media/Player.tsx

```diff
@@ -69,26 +69,7 @@ export type PlayerProps = {
 
   /** The refetch interval for the playback info hook (used with `playbackId` to query until there is a valid playback URL) */
   refetchPlaybackInfoInterval?: number;
-} & (
-  | {
-      src: string | string[] | null | undefined;
```

Comment:
Ah thanks! I did have issues to get typescript to like what I was setting here. Wdyt of having only the requirement of having an src or playbackId? Just so it complains if you don't set anything. something like
```ts
& ({src: string|...} | {playbackId: string|...})
```

Reply:
Addressed in https://github.com/livepeer/livepeer.js/commit/703ff7552e79a6e9994439cb05a1446c6e5037f4